doc/build_linux.md

```diff
@@ -1,136 +1,80 @@
-         
-         `sudo update-ca-certificates -f` (http://stackoverflow.com/a/29313285/126823)
-         
-    2. On Ubuntu 14.04:  
+2. Install dependencies. The scripts/docker directory contains build files that contain commands to install all required dependencies for various Linux flavors. Please consult these if you are having problems installing.
+
+    1. On Ubuntu 16.04:  
     
-         `sudo apt-get install build-essential git cmake cmake-qt-gui libboost-all-dev libpython2.7-dev lua5.1 liblua5.1-0-dev openjdk-7-jdk swig libxerces-c-dev doxygen xsltproc libav-tools python-tk python-imaging-tk libbz2-dev`  
+         `sudo apt-get install build-essential git libboost-all-dev libpython2.7-dev openjdk-8-jdk swig doxygen xsltproc ffmpeg python-tk python-imaging-tk`  
          
-         `export JAVA_HOME=/usr/lib/jvm/java-7-openjdk-amd64/`  
+         `export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64/`
          
          `sudo update-ca-certificates -f` (http://stackoverflow.com/a/29313285/126823)
          
     3. On Debian 8:  
     
-         `sudo apt-get install build-essential git cmake cmake-qt-gui libboost-all-dev libpython2.7-dev lua5.1 liblua5.1-0-dev openjdk-7-jdk swig xsdcxx libxerces-c-dev doxygen xsltproc libav-tools python-tk python-imaging-tk`  
+         `sudo apt-get install build-essential git libboost-all-dev libpython2.7-dev openjdk-8-jdk swig doxygen xsltproc libav-tools python-tk python-imaging-tk`  
          
-         `export JAVA_HOME=/usr/lib/jvm/java-7-openjdk-amd64/`  
+         `export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64/`  
          
          `sudo update-ca-certificates -f` (http://stackoverflow.com/a/29313285/126823)
          
-    4. On Debian 7:  
-    
-         `sudo apt-get install build-essential git cmake cmake-qt-gui libbz2-dev python2.7-dev lua5.1 liblua5.1-0-dev openjdk-7-jdk swig libxerces-c-dev doxygen xsltproc ffmpeg python-tk python-imaging-tk`  
-         
-         `export JAVA_HOME=/usr/lib/jvm/java-7-openjdk-amd64/`  
-         
-         Also, remove Java 6 and make sure that `java -version` returns the right version (1.7).  
-         
-         `sudo update-ca-certificates -f` (http://stackoverflow.com/a/29313285/126823)
-
-    5. On Fedora 23:  
+    5. On Fedora 26:  
 
         `su -c 'dnf install http://download1.rpmfusion.org/free/fedora/rpmfusion-free-release-$(rpm -E %fedora).noarch.rpm http://download1.rpmfusion.org/nonfree/fedora/rpmfusion-nonfree-release-$(rpm -E %fedora).noarch.rpm'` (for ffmpeg)  
 
-        `sudo dnf install git cmake cmake-gui boost-devel python-devel java-1.8.0-openjdk-devel swig xsd xerces-c-devel doxygen libxslt ffmpeg gcc-c++ mono-devel compat-lua compat-lua-devel lua-socket-compat tkinter python-pillow-tk`
+        `sudo dnf install git boost-devel python-devel java-1.8.0-openjdk-devel swig doxygen libxslt ffmpeg gcc-c++ tkinter python-pillow-tk`
     
-    6. On Arch Linux:
```

Comment:
Would be nice to keep the Arch documentation - it was a community contribution; we could ask the original contributor to update it if we don't have the bandwidth to update/test this ourselves.

Reply:
Left it marked as likely out of date,